timelock-impl/src/test/java/com/palantir/atlasdb/timelock/adjudicate/FeedbackAnalysisTest.java

```diff
@@ -0,0 +1,291 @@
+        return getClientFeedbackReport(serviceName,
+                nodeId,
+                Constants.MIN_REQUIRED_LEADER_TIME_ONE_MINUTE_RATE + 1,
+                Constants.MAX_ACCEPTABLE_LEADER_TIME_P99_MILLI - 1,
+                Constants.MIN_REQUIRED_START_TXN_ONE_MINUTE_RATE + 1,
+                Constants.MAX_ACCEPTABLE_START_TXN_P99_MILLI - 1);
+    }
+
+    private ConjureTimeLockClientFeedback getUnknownClientFeedbackReport(String serviceName, UUID nodeId) {
+        return getClientFeedbackReport(serviceName,
+                nodeId,
+                Constants.MIN_REQUIRED_LEADER_TIME_ONE_MINUTE_RATE - 1,
+                Constants.MAX_ACCEPTABLE_LEADER_TIME_P99_MILLI - 1,
+                Constants.MIN_REQUIRED_START_TXN_ONE_MINUTE_RATE - 1,
+                Constants.MAX_ACCEPTABLE_START_TXN_P99_MILLI - 1);
+    }
+
+    private ConjureTimeLockClientFeedback getReportWithLeaderTimeMetricInUnknownState(String serviceName, UUID nodeId) {
+        return getClientFeedbackReport(serviceName,
+                nodeId,
+                Constants.MIN_REQUIRED_LEADER_TIME_ONE_MINUTE_RATE - 1,
+                Constants.MAX_ACCEPTABLE_LEADER_TIME_P99_MILLI - 1,
+                Constants.MIN_REQUIRED_START_TXN_ONE_MINUTE_RATE + 1,
+                Constants.MAX_ACCEPTABLE_START_TXN_P99_MILLI - 1);
+    }
+
+    private ConjureTimeLockClientFeedback getReportWithStartTxnMetricInUnknownState(String serviceName, UUID nodeId) {
+        return getClientFeedbackReport(serviceName,
+                nodeId,
+                Constants.MIN_REQUIRED_LEADER_TIME_ONE_MINUTE_RATE + 1,
+                Constants.MAX_ACCEPTABLE_LEADER_TIME_P99_MILLI - 1,
+                Constants.MIN_REQUIRED_START_TXN_ONE_MINUTE_RATE - 1,
+                Constants.MAX_ACCEPTABLE_START_TXN_P99_MILLI - 1);
+    }
+
+    private ConjureTimeLockClientFeedback getReportWithLeaderTimeMetricInUnhealthyState(String serviceName, UUID nodeId) {
+        return getClientFeedbackReport(serviceName,
+                nodeId,
+                Constants.MIN_REQUIRED_LEADER_TIME_ONE_MINUTE_RATE + 1,
+                Constants.MAX_ACCEPTABLE_LEADER_TIME_P99_MILLI + 1,
+                Constants.MIN_REQUIRED_START_TXN_ONE_MINUTE_RATE + 1,
+                Constants.MAX_ACCEPTABLE_START_TXN_P99_MILLI - 1);
+    }
+
+    private ConjureTimeLockClientFeedback getReportWithStartTxnMetricInUnHealthyState(String serviceName, UUID nodeId) {
+        return getClientFeedbackReport(serviceName,
+                nodeId,
+                Constants.MIN_REQUIRED_LEADER_TIME_ONE_MINUTE_RATE + 1,
+                Constants.MAX_ACCEPTABLE_LEADER_TIME_P99_MILLI - 1,
+                Constants.MIN_REQUIRED_START_TXN_ONE_MINUTE_RATE + 1,
+                Constants.MAX_ACCEPTABLE_START_TXN_P99_MILLI + 1);
+    }
+
+    private ConjureTimeLockClientFeedback getClientFeedbackReport(String serviceName, UUID nodeId,
+            int i1, int i2, int i3, int i4) {
```

Comment:
nit: I know this is extracted but we should probably name the parameters to be what they are